.gitignore

```diff
@@ -1 +1,3 @@
 *.sw?
+parallel-extensions/*.crx
+parallel-extensions/*.zip
```

Comment:
Now that I'm testing in both browsers, I realized that Firefox packaged extensions are actually XPIs ...

Reply:
That's how I've been doing it on my local machine, but figured you must know something I don't and have a differently configured environment that made this work 😆updating now to make these changes to the `.gitignore` and the `README.md`

Reply:
Ok, updated :+1:

Reply:
>figured you must know something I don't

Please ask next time! You want to understand and double-check the changes you make. You don't want to assume.